src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -1293,7 +1297,7 @@ public void setActionsServerBaseUri(String actionsServerBaseUri) {
         return Mono.fromCallable(() -> {
             StudyEntity studyEntity = new StudyEntity(uuid, userId, studyName, LocalDateTime.now(ZoneOffset.UTC), networkUuid, networkId, description, caseFormat, caseUuid, casePrivate, isPrivate, loadFlowStatus, loadFlowResult, null, loadFlowParameters, securityAnalysisUuid);
             return studyRepository.save(studyEntity);
-        });
+        }).doOnNext(studyEntity -> networkModificationTreeService.createRoot(studyEntity));
```

Comment:
do we want to do everything in one transaction ? it would recover better from errors